frappe/core/doctype/file/file.py

```diff
@@ -140,6 +141,26 @@ def validate(self):
 		if self.file_url and (self.is_private != self.file_url.startswith('/private')):
 			frappe.throw(_('Invalid file URL. Please contact System Administrator.'))
 
+	def validate_attachment_limit(self):
```

Comment:
Something in this function is causing / forcing a 5 file attachment limit for all doctypes.